src/main/java/racingcar/view/Input.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+import racingcar.utils.ValidationUtils;
+
+public class Input {
+  private static final String INPUT_CAR_NAMES_MESSAGE = "자동차 이름을 입력하세요. (이름은 쉼표(,) 기준으로 구분)";
```

Comment:
구분자로 ,를 사용하셨는데, 위에서 드린 코멘트처럼 `,`를 상수로 분리한다면 상수화한 문장도 수정해볼 수 있을 것 같아요.
```suggestion
  private static final String INPUT_CAR_NAMES_MESSAGE = String.format("자동차 이름을 입력하세요. (이름은 쉼표(%s) 기준으로 구분)", DELEMITER);
```